helm-charts/basehub/values.yaml

```diff
@@ -64,14 +64,14 @@ staticWebsite:
       privateKey: ""
 
 nfs:
-  enabled: true
+  enabled: false
   dirsizeReporter:
     enabled: true
   shareCreator:
     enabled: true
     tolerations: []
   pv:
-    enabled: true
+    enabled: false
```

Comment:
Since this switches these from true to false, I'm updating all cluster config explicitly configuring nfs.